src/questions/chapter5.yaml

```diff
@@ -0,0 +1,1239 @@
+- uuid: bd6b88a6-ea79-4745-8a9e-3fcccf36a7ab
+  type: MultipleChoice
+  content: The feature in object-oriented programming that allows the same operation to be carried out differently, depending on the object, is 
+  hint: noHint
+  explanation:
+  answerKeys:
+    - B
+  topic: OOP
+  subTopics:
+    - Polymorphism
+  difficultyLevel: Low
+  choices:
+    - content: Inheritance
+      key: A
+      order: 0
+    - content: Polymorphism
+      key: B
+      order: 1
+    - content: Overriding
+      key: C
+      order: 2
+    - content: Overloading
+      key: D
+      order: 3
+
+- uuid: f02bba8b-1836-4052-abdb-cb58f6f30554
+  type: MultipleChoice
+  content: 
+    Consider the following two statements 
+    * (a)A publicly derived class is a subtype of its base class. 
+    * (b)Inheritance provides for code reuse.
+  hint: noHint
+  explanation: 
+    - A publicly derived class is a subtype of its base class.
+    - Inheritance provides for code reuse.  
+  answerKeys:
+    - A
+  topic: OOP
+  subTopics:
+    - Inheritance
+  difficultyLevel: Low
+  choices:
+    - content: Both the statements (a) and (b) are correct.
+      key: A
+      order: 0
+    - content: Statements (a) is correct and statement (b) is incorrect.
+      key: B
+      order: 1
+    - content: Statements (a) is incorrect and statement (b) is correct.
+      key: C
+      order: 2
+    - content: Both the statements (a) and (b) are incorrect.
+      key: D
+      order: 3
+
+- uuid: 4c08743a-6381-4508-9575-bd315a9dd1d0
+  type: MultipleChoice
+  content: Abstraction and encapsulation are fundamental principles that underlie the object oriented approach to software development. What can you say about the following two statements ? 
+    * I. Abstraction allows us to focus on what something does without considering the complexities of how it works. 
```

Comment:
`Abstraction allows us to focus on what something does without considering the complexities of how it works.` this statement is true to a certain extent. We need to reword it so that is wrong in all contexts  like
`Abstraction talks about bundling data and methods together in one Contract`.  etc.